dkpro-core-stanfordnlp-gpl/src/main/java/de/tudarmstadt/ukp/dkpro/core/stanfordnlp/StanfordPosTaggerTrainer.java

```diff
@@ -91,19 +91,20 @@ public void initialize(UimaContext aContext)
         super.initialize(aContext);
         
         try {
-            String p = clusterFile.getAbsolutePath();
-            if (p.contains("(") || p.contains(")") || p.contains(",")) {
-                // The Stanford POS tagger trainer does not suppor these characters in the cluster
-                // files path. If we have those, try to copy the clusters somewhere save before
-                // training. See: https://github.com/stanfordnlp/CoreNLP/issues/255
-                File tempClusterFile = File.createTempFile("dkpro-stanford-pos-trainer",
-                        ".cluster");
-                FileUtils.copyFile(clusterFile, tempClusterFile);
-                clusterFile = tempClusterFile;
-                clusterFilesTemporary = true;
-            }
-            else {
-                clusterFilesTemporary = false;
+            clusterFilesTemporary = false;
+
+            if (clusterFile != null) {
+                String p = clusterFile.getAbsolutePath();
+                if (p.contains("(") || p.contains(")") || p.contains(",")) {
+                    // The Stanford POS tagger trainer does not support these characters in the cluster
+                    // files path. If we have those, try to copy the clusters somewhere save before
+                    // training. See: https://github.com/stanfordnlp/CoreNLP/issues/255
+                    File tempClusterFile = File.createTempFile("dkpro-stanford-pos-trainer",
+                            ".cluster");
+                    FileUtils.copyFile(clusterFile, tempClusterFile);
+                    clusterFile = tempClusterFile;
+                    clusterFilesTemporary = true;
+                }
```

Comment:
Are these changes functionally relevant? It seems to me that the only difference is that `clusterFilesTemporary = false` has been pulled out from the `else` branch and moved in front of the `if`-clause.